apps/dav/lib/CardDAV/CardDavBackend.php

```diff
@@ -933,11 +933,17 @@ private function readBlob($cardData, &$modified = false) {
 			$cardData = stream_get_contents($cardData);
 		}
 
-		$cardDataArray = explode("\r\n", $cardData);
+		// Micro optimisation
+		// don't loop through
+		if (strpos($cardData, 'PHOTO') === 0) {
```

Comment:
To micro optimize a bit more maybe:

```suggestion
		if (strpos($cardData, 'PHOTO:data:image/') === 0) {
```